packages/fast-animation/package.json

```diff
@@ -2,16 +2,17 @@
   "name": "@microsoft/fast-animation",
   "description": "An animation library that simplifies interactions and animations using the Web Animation API.",
   "version": "1.8.0",
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
   "license": "MIT",
-  "main": "dist/index.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
-  "author": {
-    "name": "Microsoft",
-    "email": "fwq@microsoft.com"
-  },
+  "readme": "README.md",
```

Comment:
I don't believe this is a field that is part of the package.json [spec](https://docs.npmjs.com/files/package.json). I also think README.md at the root of a package is the default.